From: Rene Kita <mail@rkta.de>
Date: Fri Dec 22 19:15:02 CET 2023
Subject: Remove unused argument check

newFileStream is never called with NULL as second argument, drop the
check and make the argument necessary.
---
 istream.c |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

--- a/istream.c
+++ b/istream.c
@@ -116,10 +116,7 @@ newFileStream(FILE * f, void (*closep) (
     init_base_stream(&stream->base, STREAM_BUF_SIZE);
     stream->file.type = IST_FILE;
     stream->file.handle = f;
-    if (closep)
-	stream->file.close = closep;
-    else
-	stream->file.close = (void (*)())fclose;
+    stream->file.close = closep;
     stream->file.read = (int (*)())file_read;
     return stream;
 }
